src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -175,6 +175,7 @@
 
 			@RequestParam(required = false) String ecl,
 			@RequestParam(required = false) String statedEcl,
+			@RequestParam(required = false) Relationship.CharacteristicType form,
```

Comment:
I'm a bit concerned that it's not clear that setting the "form" parameter will cause the endpoint to run slightly more slowly because of the additional fetch of the leaf flag. Since this is supposed to be the fast concept search endpoint (as opposed to the /browser one) I think requesting the leaf flag explicit.

- Please could you give "form" a default value of "inferred" please.
- Could you also add an "includeLeafFlag" boolean parameter that defaults to false and triggers the additional fetch when set to true.

Once this PR is accepted I will have a think about aligning the "ecl" and "statedEcl" parameters with the new "form" param.